addresses.json

```diff
@@ -183,11 +183,11 @@
     "CollectionFactory": "0x2A72Ec4241Ac4fBc915ae98aC5a5b01AdE721f4B",
     "CollectionStore": "0x6ddF1b1924DAD850AdBc1C02026535464Be06B0c",
     "Marketplace": "0x2a09a000224f5dbe0E17214cA95CDe506DA7CB74",
-    "TPR": "0x83cB8cd93698c0726AF6b5497540C9A1B410fFA2",
+    "TPR": "0x640969Bfa2feA201Aeb9f482E5DdddC25586e82c",
```

Comment:
Can we add the TPRProxy and the implementation in the TPR prop?

Reply:
Good idea, added!